core_impl/src/main/java/io/opencensus/implcore/stats/ViewManagerImpl.java

```diff
@@ -38,6 +42,17 @@ public ViewData getView(View.Name viewName) {
     return statsManager.getView(viewName);
   }
 
+  @Override
+  public Set<View> getAllExportedViews() {
```

Comment:
The way how I think this will be used is by querying this every X seconds before pulling the data (in the exporter) correct? Probably worth having a cached version of this result in the StatsManager:

private volatile List<View> exportedViews;  // gets updated all the time when register/unregister a view and you have the sync there so this will be consistent.

Reply:
I cached the set of views and recalculated them when they are needed.  It does add some complexity, though.